app/models/host/discovered.rb

```diff
@@ -157,10 +157,10 @@ def reboot
     msg = status ? 'successful' : 'failed'
     logger.info "ForemanDiscovery: reboot result: #{msg}"
     status
-    rescue => e
-      logger.info "ForemanDiscovery: reboot result: failed"
-      logger.warn e.backtrace.join('\n')
-      raise e
```

Comment:
Removing this seems like a bad idea, since you'll also affect the UI outside of orchestration by not returning a reason for the failure.  Better to deal with it in orchestration if you want to change the behaviour.